.travis/build-and-test-set-1.sh

```diff
@@ -9,9 +9,9 @@
   "project vector" test \
   "project vectortile" test \
   "project gdal" test \
-  "project geowave" compile test:compile \
+  # "project geowave" compile test:compile \
```

Comment:
I think the right change here is to rename all `-212` files into files without this postfix and all files wihtout postfix to rename to `-211` files and add pass everywhere an explicit scala version